torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -257,8 +257,14 @@ void PropagateCatShape(Node * cat_node) {
     }
     return false;
   };
-  auto list_node = cat_node->namedInput(attr::tensors)->node();
-  if (list_node->kind() == prim::ListConstruct) {
+  if (cat_node->kind() == prim::FusedConcat) {
```

Comment:
Is this necessary? `tensors` go out of scope right away, does `cat_node->inputs()` have some side effects? I would hope not.